packages/espresso/android/src/main/java/androidx/test/espresso/flutter/exception/AmbiguousWidgetMatcherException.java

```diff
@@ -13,6 +13,8 @@
 public final class AmbiguousWidgetMatcherException extends RuntimeException
     implements EspressoException {
 
+  private static final long serialVersionUID = 9000L;
```

Comment:
I had warnings about the missing `serialVersionUID`, adding it in fixes the warning. I chose the values arbitrarily though.
Any ideas on better values?

Reply:
Since I can't imagine a scenario where we would serialize this, I don't think it matters. I would suggest just using `0L` rather than a completely arbitrary number.